src/main/java/redis/clients/jedis/commands/ACLCommands.java

```diff
@@ -0,0 +1,30 @@
+package redis.clients.jedis.commands;
+
+import redis.clients.jedis.UserACL;
+
+import java.util.List;
+
+public interface ACLCommands {
```

Comment:
How about `AccessControlListCommands`/`AccessControlCommands`?

Reply:
I have solved that in the commit 439d7aa692cd0ea4116d6307eefac2a76f5ca5f0

Reply:
I have removed this class, and as you suggested I have moved the AdvancedJedisCommands & AdvancedBinaryJedisCommands interfaces